src/Compilers/Core/Portable/SourceGeneration/IncrementalGeneratorInitializationContext_ForAttributeWithSimpleName.cs

```diff
@@ -0,0 +1,284 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Transactions;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.SourceGeneration;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis;
+
+using Aliases = ArrayBuilder<(string aliasName, string symbolName)>;
```

Comment:
generally, because if it's before yo have to fully qualify all type names within it.  e.g. this would have to be: `using Aliases = Microsoft.CodeAnalysis.PooledObjects.ArrayBuilder<(string aliasName, string symbolName)>;`

It gets worse if the inner types need to be qualified as well.  So my general pattern is: normal usings on the outside, aliases on the inside.